localgov_base.libraries.yml

```diff
@@ -124,6 +124,10 @@ topic-list-builder:
     theme:
       css/components/topic-list-builder.css: {}
 
+guide:
```

Comment:
should this be 'guides'? or else change in the attach_library so that it matches?